model/Software/Vocabularies/SBOMDocumentTypes.md

```diff
@@ -0,0 +1,25 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# SBOMDocumentTypes
+
+## Summary
+
+Provides a set of values to be used to describe the common types of SBOMs that tools may create.
+
+## Description
+
+This list of SBOM types with definitions, as defined in [Types of Software Bill of Material (SBOM) Documents](https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf), published on April 21, 2023. 
```

Comment:
This sentence is not finished. Maybe a "The" at the beginning?
I like the word "set" used in the summary, that maybe clarifies the issue that this is not a list in the sense of properties having multiple values.

Reply:
@armintaenzertng - have adjusted the sentence,  please re-review